python/pyarrow/tests/test_cffi.py

```diff
@@ -414,6 +414,37 @@ def test_export_import_batch_reader(reader_factory):
         pa.RecordBatchReader._import_from_c(ptr_stream)
 
 
+@needs_cffi
+def test_export_import_exception_reader():
+    c_stream = ffi.new("struct ArrowArrayStream*")
```

Comment:
Can you add a reference to the GH issue?

Reply:
Done.